apps/search-poc/src/containers/__tests__/FileDownLoad.test.tsx

```diff
@@ -0,0 +1,68 @@
+import * as renderer from "react-test-renderer";
+import FileDownload from '../FileDownLoad';
+import React from "react";
+import {  mount } from "enzyme";
```

Comment:
We could use single quotes here to be consistent, also here is an extra space before 'mount'